src/tests.rs

```diff
@@ -1389,7 +1391,7 @@ fn exercise_full_api<'a>() {
         ::nodes::NodeValue::Text(text) => {
             let _: &Vec<u8> = text;
         }
-        ::nodes::NodeValue::TaskItem(checked) => {
+        ::nodes::NodeValue::TaskItem { checked, .. } => {
             let _: &bool = checked;
```

Comment:
Since we're adding to the API, let's add a similar "test" for the `symbol` field here.